dashboard/js/publishArtifact.js

```diff
@@ -419,26 +426,26 @@ function publishArtifact(){
 			var distributor = $(mediaType + ' #distributor').val();	
 
 			if (!isBlank(director))
-				alexandriaMedia["info"]["extra-info"]["artist"] = director;
+				alexandriaMedia["info"]["extraInfo"]["artist"] = director;
 
 			if (!isBlank(distributor))
-				alexandriaMedia["info"]["extra-info"]["company"] = distributor;
+				alexandriaMedia["info"]["extraInfo"]["company"] = distributor;
 		} else if (mediaType == '#thing'){
 			//################################
 			//            THING
 			//################################
 			var creatorName = $(mediaType + ' #creatorName').val();
 
 			if (!isBlank(creatorName))
-				alexandriaMedia["info"]["extra-info"]["creator"] = creatorName;
+				alexandriaMedia["info"]["extraInfo"]["creator"] = creatorName;
 		} else if (mediaType == '#html'){
 			//################################
 			//             HTML
 			//################################
 			var creatorName = $(mediaType + ' #creatorName').val();
 
 			if (!isBlank(creatorName))
-				alexandriaMedia["info"]["extra-info"]["creator"] = creatorName;
+				alexandriaMedia["info"]["extraInfo"]["creator"] = creatorName;
```

Comment:
[todo: future point] Phew, glad that's over. There must be a way of doing this kind of repeated check and assignment.

```
if (!isBlank(creatorName))
  alexandriaMedia["info"]["extraInfo"]["creator"] = creatorName;
```